src/SFA.DAS.EmployerFinance/Configuration/HostEnvironmentSettings.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.Extensions.Logging;
+
+namespace SFA.DAS.EmployerFinance.Configuration
+{
+    public struct HostEnvironmentSettings
+    {
+        public string EnvironmentName { get; set; }
+        public string AppSettingsFilePath { get; set; }
+        public LogLevel MinLogLevel { get; set; }
```

Comment:
It looks like we're setting this in two places:

* `appsettings.json
* nlog.config (if this is set to `Debug` it will only lof `Info` if that's what's set in `appsettings.json`)

Could just the nlog.config value be used instead?

Reply:
ok i'll see if removing the min log level call works and then we don't need the environment settings at all